### PR TITLE
add the Python Visualizer and the Canadian Computing Competition

### PR DESCRIPTION
**Pull request by fffelix-jan**

### CODE REVIEW

**Comment by fffelix-jan** [comment]
Also, does my contribution count towards the Hacktoberfest?

File: README.md [hunk 1]
```diff
@@ -638,6 +639,7 @@ When learning CS, there are some useful sites you must know to get always inform
 - [Topic Wise Problem For Competitive Programmer](https://a2oj.com/categories) : Topic wise Practise Problem
 - [Archived Problems - Project Euler](https://projecteuler.net/archives) : Problems Archives
 - [Art of Problem Solving](https://artofproblemsolving.com) : Is math class too easy for you? You've come to the right place!
+- [Canadian Computing Competition](https://cccgrader.com/) : Coding competition held in many Canadian high schools. Presented by the University of Waterloo.
```
Comment (sdmg15) [inline README.md:642]: We might not need this for now, the pythontutor link is enough. Please can you update accordingly ?

Comment (fffelix-jan) [reply README.md]: OK, removed!

**Comment by fffelix-jan** [comment]
It's ready to merge now!

**Pull request merged**